package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "start": "node ./app/app.js",
     "lint": "eslint ./app/*/*.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "up": "docker-compose up -d --build",
```

Comment:
Could we not detach by default? Ctrl + P + Q is detaching, but to attach back it's a bit more cumbersome.